### PR TITLE
Qmaps-824 zoom fixes

### PR DESCRIPTION
**Pull request by xem**

### CODE REVIEW

File: src/adapters/poi/idunn_poi.js [hunk 1]
```diff
@@ -58,6 +58,9 @@ export default class IdunnPoi extends Poi {
       if(err === 404) {
         return
       }
+      else if(err === 400){
```
Comment (GuillaumeGomez) [inline src/adapters/poi/idunn_poi.js]: Why not combining this condition with the previous one considering there is no specific treatment?

File: src/panel/category_panel.js [hunk 2]
```diff
@@ -52,15 +52,15 @@ export default class CategoryPanel {
   }
 
   async search() {
-    this.loading = true
+    this.loading = true;
```
Comment (GuillaumeGomez) [inline src/panel/category_panel.js]: Just curiosity: why adding the semicolon when there are none in the rest of the code?

File: src/panel/category_panel.js [hunk 3]
```diff
@@ -52,15 +52,15 @@ export default class CategoryPanel {
...
     let bbox = window.map.bbox()
     let urlBBox = [bbox.getWest(),bbox.getSouth(),bbox.getEast(),bbox.getNorth()]
       .map((cardinal) => cardinal.toFixed(7))
       .join(',')
 
-    this.pois = await IdunnPoi.poiCategoryLoad(urlBBox, 50, this.categoryName)
-    
-    this.loading = false
+    this.pois = await IdunnPoi.poiCategoryLoad(urlBBox, 50, this.categoryName, this)
+    this.loading = false;
```
Comment (GuillaumeGomez) [inline src/panel/category_panel.js]: Same question.

File: src/panel/category_panel.js [hunk 4]
```diff
@@ -82,6 +82,23 @@ export default class CategoryPanel {
     this.search()
     await this.panel.update()
     UrlState.pushUrl()
+
+    // Apply minimal zoom when opening a category
+
+    // Zoom < 5: focus on Paris
+    if(window.map.mb.getZoom() < 5){
```
Comment (GuillaumeGomez) [inline src/panel/category_panel.js]: Wouldn't a `switch` be more appropriate in here? It'd allow to call `window.map.mb.getZoom()` only once.

File: src/panel/category_panel.js (hunk 2 above)
Comment (xem) [reply src/panel/category_panel.js]: sorry, reflex

File: src/panel/category_panel.js (hunk 4 above)
Comment (xem) [reply src/panel/category_panel.js]: I agree to calling getZoom only once, but not for the switch. Switches are not ideal to test < / > conditions...  right?

**Pull request merged**